src/styles/partials/components/_icon-button.scss

```diff
@@ -29,3 +29,65 @@
 		background-color: $vibrant-blue;
 	}
 }
+.dg_icon-buttonDefault {
```

Comment:
the class should be {namespace}_{component}--{modifier} so in this case, `.dg_icon-button--default`